cmd/flux/reconcile_with_source.go

```diff
@@ -80,10 +81,15 @@ func (reconcile reconcileWithSourceCommand) run(cmd *cobra.Command, args []strin
 		reconciliationHandled(ctx, kubeClient, namespacedName, reconcile.object, lastHandledReconcileAt)); err != nil {
 		return err
 	}
-	logger.Successf("%s reconciliation completed", reconcile.kind)
 
-	if !apimeta.IsStatusConditionTrue(*reconcile.object.GetStatusConditions(), meta.ReadyCondition) {
-		return fmt.Errorf("%s reconciliation failed", reconcile.kind)
+	readyCond := apimeta.FindStatusCondition(*reconcile.object.GetStatusConditions(), meta.ReadyCondition)
+	if readyCond == nil {
+		return fmt.Errorf("status can't be determined")
+	}
+
+	logger.Successf("%s reconciliation completed", reconcile.kind)
```

Comment:
I don't think we need this anymore.

Reply:
@SomtochiAma you need to fix the test now and remove this log from the golden file.